airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -1,2 +1,29 @@
-// Initialize the editor with the div id and path to Popcorn Editor.
-PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+$(document).ready(function () {
+  function getSlug() {
+    var config = $('slug');
+    if (config.length) {
+      return config.data('url');
+    } else {
+      console.error('Slug not provided in DOM. Cannot get slug');
+    }
+  }
+
+  PopcornEditor.listen(PopcornEditor.events.loaded, function () {
+    $.get('/popcorn/data/', {
```

Comment:
I'm not quite sure how this would be kept in `urls.py` the path to the editor.html file is _strictly_ related to the PopcornEditor.js code. I could hard code it into the template file but that also seems to disconnect it from the actual code that it is semantically linked to.